tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
```

Comment:
All of these might not be necessary, I was working through which ones were needed or not but starting from a point where at least it was all working for us.